addons/jest/src/components/Result.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import { styled } from '@storybook/theming';
+
+// @ts-ignore TODO: migrate Message to typescript
```

Comment:
`// @ts-ignore TODO: migrate Message to typescript` it looks like you did it 😉

Reply:
🎉